fe/fe-core/src/main/java/com/starrocks/connector/hudi/HudiRemoteFileIO.java

```diff
@@ -83,6 +87,18 @@ public Map<RemotePathKey, List<RemoteFileDesc>> getRemoteFiles(RemotePathKey pat
             HoodieTableFileSystemView fileSystemView = new HoodieTableFileSystemView(metaClient,
                     timeline, statuses.toArray(new FileStatus[0]));
             String queryInstant = latestInstant.get().getTimestamp();
+            if (ConnectContext.get() != null) {
```

Comment:
This is not compatible with metadata cache. Since metadata cache will cache file description for former queried partitions. The work around is add timestamp to `RemotePathKey`. @stephen-shelby Do you have any suggestion.